analyses/fusion_filtering/references/genelistreference.txt

```diff
@@ -6752,3 +6752,7 @@
 "6751"	"GAK"	"PfamKinase"	"Kinase"
 "6752"	"PIK3C2B"	"PfamKinase"	"Kinase"
 "6753"	"HUNK"	"PfamKinase"	"Kinase"
+"6754"	"IGH@"	"addedToallOnco_Feb2017.tsv"	"Oncogene"
```

Comment:
I think handling this way is fine for now. However, the better solution/design decision is to deal with these atypical or inconsistent gene symbols in the _standardization steps_. I can imagine a situation where you have some reference file that essentially contains genes that need to be recoded and what they should be standardized to for each caller you support.

Reply:
Thanks for reviewing! Yes, I agree that would be a better idea going forward, I will add that as a future PR.